sklearn/preprocessing/data.py

```diff
@@ -1921,6 +1928,12 @@ def fit(self, K, y=None):
         K : numpy array of shape [n_samples, n_samples]
             Kernel matrix.
 
+        K_fit_rows_ : numpy array of shape [1, n_samples]
```

Comment:
Thanks for the PR! these are in the wrong place, they should be in the class documentation under attributes, not in ``fit``.

Reply:
ah, sorry about the mistake, fixing now